samples/contrib/aws-samples/debugger_component_demo/debugger-component-demo.py

```diff
@@ -11,19 +11,42 @@
 from kfp import dsl
 from kfp.aws import use_aws_secret
 
+sagemaker_train_op = components.load_component_from_file(
+    "../../../../components/aws/sagemaker/train/component.yaml"
+)
 
-cur_file_dir = os.path.dirname(__file__)
-components_dir = os.path.join(cur_file_dir, '../../../../components/aws/sagemaker/')
 
-sagemaker_train_op = components.load_component_from_file(components_dir + '/train/component.yaml')
+def training_input(input_name, s3_uri, content_type):
+    return {
+        "ChannelName": input_name,
+        "DataSource": {"S3DataSource": {"S3Uri": s3_uri, "S3DataType": "S3Prefix"}},
+        "ContentType": content_type
+    }
+
+
+def training_debug_hook(s3_uri, collection_dict):
+    return {
+        'S3OutputPath': s3_uri,
+        'CollectionConfigurations': format_collection_config(collection_dict)
+    }
 
-debug_hook = {
-    'S3OutputPath':'s3://kubeflow-pipeline-data/mnist_kmeans_example/hook_config'
-}
 
-debug_hook['CollectionConfigurations'] = []
+def format_collection_config(collection_dict):
+    output = []
+    for key, val in collection_dict.items():
+        output.append({'CollectionName': key, 'CollectionParameters': val})
+    return output
```

Comment:
Great! We hope these methods can be used by some of our users to simplify building their own pipelines.